docs/userGuide/syntax/variables.mbdf

```diff
@@ -64,8 +64,13 @@ This is useful if you have external datasets you want to display in your site!
 To do so, assign a root variable name (`varName`) to the file path from the <tooltip content="similar to how you assign filepaths for other Nunjucks tags">root directory of the site</tooltip>. You may then access the file's variables using dot `varName.xx` or array `varName[i]` syntax, depending on the file's contents.
 
 
+{{icon_example}}
 
-{{ icon_example }}
+<box seamless> 
```

Comment:
should we remove this? a little odd since the box only contains the tabs

or shift it into the tabs maybe

Reply:
This is how it looks after removing the box. Would this be okay?

![image](https://user-images.githubusercontent.com/16506539/104806118-103baf00-5810-11eb-8b01-fa456533a059.png)